README.md

```diff
@@ -68,6 +68,13 @@ PuppetDB
   `Preferences>File Sharing` in order for these directories to be created
   and volume-mounted automatically. There is no need to add each sub directory.
 
+The default can be overridden with the `VOLUME_ROOT` environement
+variable in the `.env` file (see *Environment* below)
```

Comment:
```suggestion
variable (see *Environment* below)
```

Since env variables can be set in the `.env` file or on the command line I think we can probably just refer to it as `environment variable` without extra information on where to set it